server/middlewares/authentication.js

```diff
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken')
+
+const authenticate = (req, res, next) => {
+  try {
+    const token = req.headers.token
+    const decoded = jwt.verify(token, process.env.SECRET)
```

Comment:
verify definisiin juga function nya di helpers, biar nggak perlu require jwt lagi di sini
```suggestion
const verify = require('../helper/jwt').verify

const authenticate = (req, res, next) => {
  try {
    const token = req.headers.token
    const decoded = verify(token, process.env.SECRET)
```